collect_app/src/main/java/org/odk/collect/android/activities/FormMapActivity.java

```diff
@@ -315,18 +314,8 @@ protected void restoreFromInstanceState(Bundle state) {
         }
     }
 
-    private static int getDrawableIdForStatus(String status, boolean enlarged) {
-        switch (status) {
-            case Instance.STATUS_INCOMPLETE:
-                return enlarged ? R.drawable.ic_room_blue_48dp : R.drawable.ic_room_blue_24dp;
-            case Instance.STATUS_COMPLETE:
-                return enlarged ? R.drawable.ic_room_deep_purple_48dp : R.drawable.ic_room_deep_purple_24dp;
-            case Instance.STATUS_SUBMITTED:
-                return enlarged ? R.drawable.ic_room_green_48dp : R.drawable.ic_room_green_24dp;
-            case Instance.STATUS_SUBMISSION_FAILED:
-                return enlarged ? R.drawable.ic_room_red_48dp : R.drawable.ic_room_red_24dp;
-        }
-        return R.drawable.ic_map_point;
+    private static int getDrawableIdForStatus(boolean enlarged) {
+        return enlarged ? R.drawable.ic_room_primary_48dp : R.drawable.ic_room_primary_24dp;
```

Comment:
Could definitely be a controversial change: my thinking here is the same as with the form status icons in the form lists.